sky/engine/core/script/dart_init.cc

```diff
@@ -68,9 +68,6 @@ namespace {
 
 static const char* kDartArgs[] = {
     "--enable_mirrors=false",
-    // Dart assumes ARM devices are insufficiently powerful and sets the
-    // default profile period to 100Hz. This number is suitable for older
-    // Raspberry Pi devices but quite low for current smartphones.
```

Comment:
Its wrong. The period actually set below in 1000Hz. Also, sky engine does not care about RaspberryPi.